### PR TITLE
add_new_lims_verification_issue_template

### PR DESCRIPTION
**Pull request by idalindegaard**
### Added
- new issue template

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lims_verification_issue.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## **Description:**
+_Describe the reason for the verification and other motivations_
+
+## **Actions:**
+_Describe/list actions to perform in order to complete the verification_
+
+_In the table below, you can add the name of the new protocol(s) and write date and sign when it was created and tested in stage or production_
+| Protocol | Created in Stage | Tested in Stage | Created in Production  | Tested in Production  |
+|---|---|---|---|---|
+| | | | |
```
[the remark this answers is not in the document]
Comment (idalindegaard) [reply .github/ISSUE_TEMPLATE/lims_verification_issue.md:25]: Ah ja! Bra

File: .github/ISSUE_TEMPLATE/lims_verification_issue.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+### **Testing setup:**
+
+_If doing more extensive testing, the setup and specification and results can be documented in the table below. Don't forget to write where you document screenshots etc, if you keep them in a folder on Google Drive for example._
+
+| Protocol | Step | Test specification | Comments/Results  | P/F  | Sign | 
+|---|---|---|---|---|---| 
+|  |  |  |   |   |
```
[the remark this answers is not in the document]
Comment (idalindegaard) [reply .github/ISSUE_TEMPLATE/lims_verification_issue.md:34]: P/F kommer från Test specification, och under Test specification får man skriva vad det är man testar och vilka kriterier som gäller för att uppfylla ett godkänt test

File: .github/ISSUE_TEMPLATE/lims_verification_issue.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+_In the table below, you can add the name of the new protocol(s) and write date and sign when it was created and tested in stage or production_
+| Protocol | Created in Stage | Tested in Stage | Created in Production  | Tested in Production  |
+|---|---|---|---|---|
+| | | | |
+
+
+### **Testing setup:**
+
+_If doing more extensive testing, the setup and specification and results can be documented in the table below. Don't forget to write where you document screenshots etc, if you keep them in a folder on Google Drive for example._
```
[the remark this answers is not in the document]
Comment (idalindegaard) [reply .github/ISSUE_TEMPLATE/lims_verification_issue.md:30]: Smart!

**Comment by idalindegaard** [comment]
> Nice with a template! I had some suggestions and comments. When you feel that the template is "good enough" I think you should merge and then try it for a real verification and see if any part is missing.

@AnnaLeinfelt Right! That sounds like a good idea. In fact, all of the verifications of the workflows that we are re-creating now to be compatible with the NovaSeq X implementation are documented in cg_lims in this way (almost), but I will update the issues so they are in line with this template!

**Comment by idalindegaard** [comment]
@annaengstrom Hi, we are creating a specific issue template for LIMS verifications as we didn't have that. And auto-assigning you, if that is okay?

A verification is needed for new workflow versions or new protocols
Validations are for completely new workflows
For smaller changes like new steps or other smaller configurations, some simple tests can be done and documented as a regular issue (just as before)

File: .github/ISSUE_TEMPLATE/lims_verification_issue.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (idalindegaard) [reply .github/ISSUE_TEMPLATE/lims_verification_issue.md:34]: Jag tänker att Test specification bara kommer stå i tabellen. Eller behövs en rubrik om det ovanför tabellen också?

**Review by annaengstrom — APPROVED** [review]
This new template looks good! It can be modified futher if you notice that the template needs other changes.

**Comment by annaengstrom** [comment]
I cannot access Atlas where I am now, but please consider if any Atlas documentation needs updates, regarding LIMS validations/verifications when implementing this template.

**Comment by idalindegaard** [comment]
> I cannot access Atlas where I am now, but please consider if any Atlas documentation needs updates, regarding LIMS validations/verifications when implementing this template.

Good call! Will do that

**Review by Karl-Svard — APPROVED** [review]
Great work!

**Pull request merged**